lib/examples/compare-chromedp/README.md

```diff
@@ -0,0 +1,37 @@
+# Rod comparison with chromedp
+
+chromedp is one of the most popular libraries available for Go which controls the Chrome Devtools Protocol. 
+
+We have emulated the examples they provide using our own library so you can compare which one you prefer.
+
+Occasionally, some of these examples may break if the Rod API gets updated or if the specific websites these examples use get updated.
+We suggest you create an [issue](https://github.com/ysmood/rod/issues/new/choose).
+
+You can build and run these examples in the usual Go way:
+
+```sh
+# retrieve examples
+$ go get -u -d github.com/ysmood/rod/
+
+# run example <prog>
+$ go run $GOPATH/src/github.com/ysmood/rod/lib/examples/compare-chromedp/<prog>/main.go
+
+# build example <prog>
+$ go build -o <prog> github.com/ysmood/rod/lib/examples/compare-chromedp/<prog>/ && ./<prog>
+```
+
+| Example                   | chromedp Example                                              | Description                                                                  |
+|---------------------------|---------------------------------------------------------------|------------------------------------------------------------------------------|
+| [click](/lib/examples/compare-chromedp/click)           | [click](https://github.com/chromedp/examples/click)           | use a selector to click on an element                                        |
+| [cookie](/lib/examples/compare-chromedp/cookie)         | [cookie](https://github.com/chromedp/examples/cookie)         | set a HTTP cookie on requests                                                |
+| [emulate](/lib/examples/compare-chromedp/emulate)       | [emulate](https://github.com/chromedp/examples/emulate)       | emulate a specific device such as an iPhone                                  |
```

Comment:
All the paths from rod like `/lib/examples/compare-chromedp/emulate`.

I think it's better to use relative path here so that when we want to rename the folder we don't have to change this file again. Such as `[emulate](./emulate)`.